### DISCUSSION TITLE
GitHub deployment still showing up after deleting the project from vercel

### DISCUSSION THREAD

**Discussion by Saransh-cpp** [vercel/community#237] [Help] [answered]
I recently hosted one of my projects on vercel but ended up removing it from there, but somehow the GitHub deployment tag never goes away (clicking on which obviously leads to a 404 page). Do I need to follow some other steps or is it sufficient to delete the project from vercel to remove deployment (as per the documentation)? I'll be happy to report it as a bug if someone can confirm.

Thanks!

**Comment by kamlendras** [comment, marked as answer]
Hi,
if you are really wants remove that GitHub deployment tag delete your GitHub repository from GitHub settings and make a new repository with the previous name and add the same code.
    & another solution is that GitHub allows to transfer code from one repository to another this might help!!

**Comment by Pieparker** [comment]
The 404 page is normal – this is the case whenever a request to a Vercel project which doesn't exist (`[name].vercel.app`) is made.

To keep discussions on topic and to prevent the resurrection of old topics, this discussion has been closed. If issues persist or a future reader stumbles upon this discussion and would like to discuss it further, please create a new discussion.

**Reply by Krisztiaan** [reply]
This is not answering the question of "how do I remove the now failing deployment on GitHub", after removing Vercel from the repository?

**Reply by Saransh-cpp** [reply]
Agreed, the answer (technically a workaround) by @Kamlendrax was more correct for this question. Thanks for the help @Pieparker @Kamlendrax!
